parse_weather_data.py

```diff
@@ -0,0 +1,59 @@
+import csv
+from weather_reading import WeatherReading
+
+
+class WeatherParser:
+
+    actual_keys = {}
+
+    def parse_weather_file(self, file_path):
+        try:
+            with open(file_path, 'r') as weather_file:
+                csv_weather_file = csv.reader(weather_file)
```

Comment:
Mentioning format of file in format is not a good practice. Your assignment is specifying that something is happening with CSV so that should be enough.